src/dom/dom.js

```diff
@@ -1002,7 +997,7 @@ p5.prototype.createInput = function(value, type) {
  * function handleFile(file) {
  *   print(file);
  *   if (file.type === 'image') {
- *     img = createImg(file.data);
+ *     img = createImg(file.data, 'alt text');
```

Comment:
if this is an image not really intended to be viewed, you can use empty string `''` for alt text to indicate that to screen readers

Reply:
ohh i did not know that! that's great for this case